package.json

```diff
@@ -29,7 +26,12 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^3.0.2",
     "eslint-plugin-react": "^6.9.0",
+    "laravel-mix": "^4",
+    "path": "^0.12.7",
+    "sass": "^1.23.3",
     "sass-lint": "^1.10.2",
-    "webpack": "^2"
+    "sass-loader": "^7.3.1",
+    "vue-template-compiler": "^2.6.10",
```

Comment:
Did Laravel Mix install this? I don't think it's necessary for this project (Vue itself isn't being pulled in).

Reply:
Yes